docs/debug/provider.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
   name: linode-provider-ceph
 spec:
   package: xpkg.upbound.io/linode/provider-ceph:v0.0.3
```

Comment:
nits: we should also update this image version

Reply:
As this is only a doc, I will remove the version in place of `<insert-version>`